scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPipelineModel.scala

```diff
@@ -51,10 +52,35 @@ class H2OMOJOPipelineModel(override val uid: String)
   protected final val outputSubTypes: StringArrayParam =
     new StringArrayParam(this, "outputSubTypes", "Types of sub-columns under the output column")
 
+  // MOJO output columns describing contributions
+  protected final val outputSubColsContributions: StringArrayParam =
+    new StringArrayParam(
+      this,
+      "outputSubColsContributions",
+      "Names of contribution sub-columns under the output column")
+  protected final val outputSubTypesContributions: StringArrayParam =
+    new StringArrayParam(
+      this,
+      "outputSubTypesContributions",
+      "Types of contribution sub-columns under the output column")
```

Comment:
`the output column` -> `the contributions column`?

Reply:
Comment string was updated.